commcare_connect/opportunity/forms.py

```diff
@@ -233,7 +235,12 @@ def save(self, commit=True):
         )
         self.instance.created_by = self.user.email
         self.instance.modified_by = self.user.email
-        self.instance.organization = organization
+
+        # In the case of a managed opportunity, the organization will be null initially,
+        # as it will be set to the network manager's organization when the invitation is accepted.
+        if not self.managed_opp:
```

Comment:
This is no longer true, right? and we can remove the migration that makes org nullable?

Reply:
Yeah this is changed in the later PR's